### DISCUSSION TITLE
[Bug] -

### DISCUSSION THREAD

**Discussion by dokidokipolyamoryclub** [Monika-After-Story/MonikaModDev#9984] [QA - tech support + general]
Hello!
I'm not exactly sure what the problem is here or how to fix it and my memory of what exactly caused this 'bug' is rather hazy due
to it happening quite a few months ago.

I was messing around with the MAS game files, trying to give monika new outfits and i have no recollection of what exactly it was that I did but the game absolutely broke (can't remember what happened thereafter either, so sorry) ever since I have re-downloaded many versions of MAS and of the base game and although the base game runs fine the mod no longer works; it just doesn't react to me installing the mod anymore.

Please help! I adore MAS and really miss playing it

**Comment by Booplicate** [comment]
Make a new DDLC install and use our installer.

**Reply by dokidokipolyamoryclub** [reply]
Ive already done that a few times and it hasn't worked, both with a new version of ddlc and of MAS

**Reply by Booplicate** [reply]
Define not working. What exactly happens? Does the installer tell you you selected the wrong directory?

**Reply by dokidokipolyamoryclub** [reply]
the game just has no reaction at all to the mod being installed

**Comment by dokidokipolyamoryclub** [comment]
<img width="432" alt="Screen Shot 2023-05-03 at 8 52 07 pm" src="https://user-images.githubusercontent.com/132450725/235896546-2379f67e-6789-43a7-ba06-5d61e80a30c0.png">
![Uploading Screen Shot 2023-05-03 at 8.52.32 pm.png…]()

**Reply by dokidokipolyamoryclub** [reply]
The file is where it's meant to be and yet the game still displays its ending screen and doesn't react at all

**Reply by Booplicate** [reply]
Use the installer, you did it manually.

**Reply by dokidokipolyamoryclub** [reply]
<img width="689" alt="Screen Shot 2023-05-03 at 8 56 27 pm" src="https://user-images.githubusercontent.com/132450725/235897446-37622f8d-1459-468b-b401-2162b777d09e.png">
 when I open the installer it just looks like this

**Reply by dokidokipolyamoryclub** [reply]
I've tried redownloading it a few times but to no avail

**Reply by Booplicate** [reply]
Because you opened it for reading, you need to run it. [Read this](https://github.com/Monika-After-Story/mas-installer/wiki/Launch-Instructions#unix-like-systems-macoslinux).

**Reply by dokidokipolyamoryclub** [reply]
Ohh, god I feel stupid now.
I need to attend to my brownies before they burn but I'll definitely try that when I get the chance! 
Thanks so much for the help
